### PR TITLE
openmpi: add latest release 3.0.2

### PR DESCRIPTION
**Pull request by floquet**
https://www.open-mpi.org/software/ompi/v3.0/:x

Signed-off-by: Daniel Topa <dantopa@lanl.gov>

### CODE REVIEW

File: var/spack/repos/builtin/packages/openmpi/package.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ class Openmpi(AutotoolsPackage):
     version('3.1.0', '0895e268ca27735d7654bf64cee6c256')  # libmpi.so.40.10.0
 
     # Still supported
+    version('3.0.2', '098fa89646f5b4438d9d8534bc960cd6')  #
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/openmpi/package.py]: Can you add a comment for which library version gets installed?

Comment (floquet) [reply var/spack/repos/builtin/packages/openmpi/package.py]: Will do.

**Comment by hppritcha** [comment]
looks like it its ```libmpi.so.40.0.2```

**Comment by floquet** [comment]
@adamjstewart: library version added

**Pull request merged**